### PR TITLE
Add aditional 3rd party tools link (mise)

### PR DESCRIPTION
**Pull request by yrsdi**
Adding mise link to release.html and patch.html

### CODE REVIEW

**Review by garazdawi — COMMENTED** [review]
Looks good, please just fix my one nit-pick and I will merge.

File: _layouts/patch.html [hunk 1]
```diff
@@ -13,14 +13,16 @@
...
-                {% if page.release < 27 %}
-                <a href="{% link /docs/{{ page.release }}/installation_guide/INSTALL.html %}">source</a> or
-                {% else %}
-                <a href="{% link /docs/{{ page.release }}/system/install.html %}">source</a> or
-                {% endif %}
-                installed using <a href="https://erlang-solutions.com/downloads">pre-built packages</a> for your OS or
-                third-party tools (such as <a href="https://github.com/kerl/kerl">kerl</a>
-                or <a href="https://github.com/asdf-vm/asdf-erlang">asdf</a>).
+                {% if page.release < 27 %} <a
+                    href="{% link /docs/{{ page.release }}/installation_guide/INSTALL.html %}">source</a> or
```
[the remark this answers is not in the document]
Comment (yrsdi) [reply _layouts/patch.html]: ok, done.

**Comment by garazdawi** [comment]
Your editor seems to be messing with html somehow. I've force pushed a commit to your branch that only adds a link to mise.

**Comment by garazdawi** [comment]
Thanks for your contribution!

**Pull request merged**